### PR TITLE
Update MOU text

### PR DESCRIPTION
**Pull request by thomasiles**
### What problem does this pull request solve?

Trello card: https://trello.com/c/VwCEnMbe/989-update-mou-with-auth0-aws-and-other-necessary-changes

Text amends and adding a link, from this document: https://docs.google.com/document/d/1Ozwggof3-NH2MW1WmlZQneDJ2BtVKkgAJmOkNEC8Tlg/edit

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: app/views/mou_signatures/_mou_version_1.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<p>Last updated: 24 October 2023</p>
+<p>Last updated: 06 November 2023</p>
```
Comment (DavidBiddle) [inline app/views/mou_signatures/_mou_version_1.html.erb:1]: Is it intentional (or if not, does it matter) that we've got 2 different date formats in this file - `06 November` here and `6 November` on line 909?

**Review by jamie-o-wilkinson — APPROVED** [review]
Looks ok to me!

File: app/views/mou_signatures/_mou_version_1.html.erb (hunk 1 above)
Comment (thomasiles) [reply app/views/mou_signatures/_mou_version_1.html.erb:1]: That's a good question - I'll stick it on the trello card

**Review by DavidBiddle — DISMISSED** [review]

File: app/views/mou_signatures/_mou_version_1.html.erb (hunk 1 above)
Comment (DavidBiddle) [reply app/views/mou_signatures/_mou_version_1.html.erb:1]: Not a blocker if you want to merge this and resolve that later if its not right

Comment (thomasiles) [reply app/views/mou_signatures/_mou_version_1.html.erb:1]: Got confirmation for the 06 format so I've updated the PR.

**Review by DavidBiddle — APPROVED** [review]

File: app/views/mou_signatures/_mou_version_1.html.erb (hunk 1 above)
Comment (DavidBiddle) [reply app/views/mou_signatures/_mou_version_1.html.erb:1]: Nice - have approved!

**Pull request merged**